homeassistant/components/tasmota/fan.py

```diff
@@ -65,6 +65,7 @@ class TasmotaFan(
     """Representation of a Tasmota fan."""
 
     _attr_supported_features = FanEntityFeature.SET_SPEED
+    _fan_speed = tasmota_const.FAN_SPEED_MEDIUM
```

Comment:
Maybe this should be `_last_known_fan_speed` instead of `_fan_speed` since it's not reset when the fan is turned off?

Reply:
I think `_fan_speed` is fine.